api/views.py

```diff
@@ -74,14 +75,17 @@ def added(request):
 def add_site(request):
     """Handle incoming POST requests to add new sites to the list.
 
-    For sites submitted via the extension.
+    Creates a new SiteRequest object for sites submitted via the extension.
 
-    Returns JSON response.
+    Returns JSON response back to the extension.
 
     """
 
     add_site = request.POST['site']
```

Comment:
May be safer to use. Not big. But good habit. 
```
add_site = request.POST.get("site", "")
if not add_site:
    data = {
         "status": "fail"
     }
else:
    ...
```
This way. If someone sends a request to this endpoint that does not have the POST data, we will not error.